app/Http/Controllers/BookingRequestController.php

```diff
@@ -151,14 +152,26 @@ public function store(CreateBookingRequest $request)
         return redirect()->route('bookings.index')->with('flash', ['banner' => 'Your Booking Request was submitted']);
     }
 
-  /**
+    public function show(BookingRequest $booking)
+    {
+        $booking->loadMissing('requester', 'reservations', 'reservations.room');
+        return inertia('Requestee/ViewBooking', [
+            'booking' => new BookingResource($booking)
+        ]);
+    }
+
+    /**
    * Show the form for editing the specified resource.
    *
    * @param BookingRequest $booking
-   * @return Response|\Inertia\Response|ResponseFactory
+   * @return RedirectResponse|Response|\Inertia\Response|ResponseFactory
    */
     public function edit(BookingRequest $booking)
     {
+        if ($booking->status == BookingRequest::REVIEW || $booking->status == BookingRequest::APPROVED ) {
```

Comment:
Could we flash a message to explain the redirect to the user plz

Reply:
I already have a message appear at the top of the view page that states that when the status is in review.

Reply:
i didn't like the flash since my solution also informs them of the blocked status without the need for a redirect in case they go straight to the view route

Reply:
that works too :)